antlr/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "cel-parser"
+description = "An antlr-based parser for the Common Expression Language (CEL)"
+repository = "https://github.com/clarkmcc/cel-rust"
+version = "0.1.0"
```

Comment:
with the name reuse, this needs bumping to at least `0.9`... wondering if we should jump to `0.10` to mark the breakage? (easier to remember, 2 digits = antlr based grammar).